dns/unbound-plus/src/opnsense/mvc/app/controllers/OPNsense/Unboundplus/Api/GeneralController.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
Can’t we use Unbound namespace?

Reply:
Sure, would better fit when it moves to core